### PR TITLE
Fix mqtt example py code which was not working

### PR DESCRIPTION
**Pull request by deekoder**
The example mqtt.py needs to be fixed for the example to work in this doc.

## Description
Updated how to subscribe to a topic and moved eg to python3

## Motivation and Context
The current example is outdated. Found it while testing something. 

## How Has This Been Tested?
Manually. 

## Types of changes

- [ X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ X ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by deekoder** [comment]
since @vadmeste  helped diagnose the rootcause adding him as reviewer.

File: docs/bucket/notifications/README.md [hunk 1]
```diff
@@ -197,33 +197,28 @@ arn:minio:sqs::1:amqp s3:ObjectCreated:*,s3:ObjectRemoved:* Filter: suffix=”.j
...
+def on_connect(client, userdata, flags, rc):
+  print("Connected with result code "+str(rc))
+  client.subscribe("minio")
 
-# The callback for when a PUBLISH message is received from the server.
 def on_message(client, userdata, msg):
     print(msg.payload)
 
 client = mqtt.Client()
+#client.connect("localhost",1883,60)
```
Comment (harshavardhana) [inline docs/bucket/notifications/README.md]: We can remove the commented code.

File: docs/bucket/notifications/README.md [hunk 2]
```diff
@@ -197,33 +197,28 @@ arn:minio:sqs::1:amqp s3:ObjectCreated:*,s3:ObjectRemoved:* Filter: suffix=”.j
...
 
-client.connect("localhost:1883", 1883, 60)
-
-# Blocking call that processes network traffic, dispatches callbacks and
-# handles reconnecting.
-# Other loop*() functions are available that give a threaded interface and a
-# manual interface.
+client.connect("localhost",1883,60)
 client.loop_forever()
+
```
Comment (harshavardhana) [inline docs/bucket/notifications/README.md]: No need for extra space here.

File: docs/bucket/notifications/README.md [hunk 3]
```diff
@@ -197,33 +197,28 @@ arn:minio:sqs::1:amqp s3:ObjectCreated:*,s3:ObjectRemoved:* Filter: suffix=”.j
 The python program below waits on mqtt topic ``/minio`` and prints event notifications on the console. We use [paho-mqtt](https://pypi.python.org/pypi/paho-mqtt/) library to do this.
 
 ```py
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (harshavardhana) [inline docs/bucket/notifications/README.md:200]: This is only true on systems which have python2 as default on my system python3 is default so `env python` works. I am assuming the code works for both python versions do we need to make this choice here?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/bucket/notifications/README.md (hunk 3 above)
Comment (deekoder) [reply docs/bucket/notifications/README.md:200]: The client library itself has python3 examples now. They have updated. I was not able to have it work with python2 on my machine without copying the library manually to site-packages. I think its best to move to python3.

File: docs/bucket/notifications/README.md (hunk 2 above)
Comment (deekoder) [reply docs/bucket/notifications/README.md]: will do

File: docs/bucket/notifications/README.md (hunk 3 above)
Comment (deekoder) [reply docs/bucket/notifications/README.md:200]: It should still work on your system if you have python3 here by calling this #! /usr/bin/env python3  even if your default python = python3. But if its not, then upon cut copy paste, it throws several errors, unless you do the above workaround (copying site-packages).

Comment (harshavardhana) [reply docs/bucket/notifications/README.md:200]: okay, so they have actively deprecated perhaps we can mention it that this example requires python3 installed.

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**